### DISCUSSION TITLE
CLOUT Split [PRE COINBASE LISTING]

### DISCUSSION THREAD

**Discussion by blackstock** [deso-protocol/dips#111] [Large CIPs]
There is a card in the "Large" task labeled "CLOUT Split". 

I was curious if this was in reference to a token split to increase the overall supply it Bitclout tokens? 

I am of the opinion that the current CLOUT price is too high and that we should prioritize increasing the supply before being listed on more exchanges. My preference would be a 1,000/1 split, taking the current price of CLOUT down to ~ $0.10/coin.

There is a strong bias in finance for retail investors to prefer cheaper tokens/coins/stocks etc. While it should be psychological, it does influence how investors view the potential upside of an asset. For example, most investors think it's easier for a token to appreciate from $0.10 -> $1 as opposed to a token appreciating from $100 -> $1,000 (both a 10x increase). We see this play out with cheap tokens all over the Crypto landscape and even in traditional finance with Penny stocks. 

I think CLOUT should be a fun, social currency and should be priced such that the majority of retail investors can buy large quantities of whole coins. And as irrational as this sounds, buying 1,000 CLOUT for $100 is more satisfying than buying 1 CLOUT for $100 - at least for a material number of investors (myself included).

**Comment by tijno** [comment]
Ive been curious about what that card is about.

Note that only a few months ago we had the opposite:

deflation bomb

which reduced supply considerably.

Clarification on the intent of that card would be welcome.

**Reply by blackstock** [reply]
To be clear on my thoughts, I'm not advocating for an inflationary currency (I'd prefer deflationary if anything). I think a one-time split to radically increase the supply would provide a better experience for retail investors so they can Invest/Tip/Buy in whole coins as opposed to fractional CLOUTs. 

And maybe this is just semantics, but I don't think a split makes Bitclout an inflationary currency. It's just a one-time event that affects everyone's holdings pro-rata and the market cap would be the same post-split.

**Reply by tijno** [reply]
I can see downsides to a 1000th reduction in value per coin. Its seen as less valuable to uninformed, and may be seen as a gimmick or trick to those more experienced but not yet onboard with the project.

There are also other complications - eg how would it affect creator coin bonding curve ?

How would it help create more balanced distribution of clout ?

**Reply by blackstock** [reply]
Cheaper CLOUT could translate to more users and adoption because users *feel* like they are early and have higher ROI potential, which would lead to a more balanced distribution. 

This isn't a new idea, the majority of top-100 projects have over 100M in circulating supply.  Of the top-100 projects, only 4 projects have a smaller circulating supply (Yearn, Maker, Compound and Kusama). 

I believe this is a limiting factor with Clout right now - it *feels* expensive for a token/coin that few people have ever heard of. 

Are you opposed to splits in general, or just a split of that magnitude?

**Reply by blackstock** [reply]
@maebeam @diamondhands0 @redpartyhat any insight you can share with this particular "Large" CIP?

**Reply by tijno** [reply]
@blackstock 

> Are you opposed to splits in general, or just a split of that magnitude?

I guess for me turning a $100 coin into a $0.01 coin feels like we want to turn bitclout into a penny stock. I dont feel thats the right value proposition for this coin.

It may help with people betting on it on exchanges - but I dont see how it helps users to actively be onboarded and using bitclout.

Eg to get started - they need sufficient balance to to 3-4 weeks of posting probably. Right now - with just 0.1 clout and they can keep going for a while.

If price is reduced 1000 fold they will need 100 clout - which could be seen as "oh my this is expensive to post".

Thats just my personal feeling about it. I appreciate that i may just be a lone voice.

However - what is important is how you think it would affect creator coin prices and the bonding curve.

Could you elaborate on that?

**Reply by blackstock** [reply]
@tijno I don't think this would impact the quantity of creator coins. It would just impact the amount of CLOUT that purchases/sales are denominated in. So, for example, instead of the first creator coin costing .1 CLOUT (or whatever it is now), maybe it costs 10 CLOUT. I'd need to look at the actual bonding curve math, but in my mind, this doesn't seem like a big refactor. 

And maybe a 100-1 split is better if you're concerned about penny stock optics. That'd bring the current CLOUT price to $1/token at the current market cap of $1B ($10/token would translate to a $10B market cap).

**Reply by tijno** [reply]
I feel its still optics that in my experience mainly day traders care about. 
So not convinced of the benefits to user adoption.
But then again I am not the norm either.

Lets see what others think.

**Comment by tijno** [comment]
In light of the recent townhall, looks like this is indeed a 100 or 100x split.

If its going to be split - then 100x should be the focus - not 1000x. 

I think 1000x will push deso into funny money / territory.

For reference - supply of top coins:

btc 21m
eth 117m
bnb 170m
sol 500m
ada 45b

Keen to understand how split would effect:

1 - creator coin bonding curve
2 - fees
3 - miner reward

Also split should be considered in light of thinking around proof of stake - it probably wont have an impact unless there was some plan around minimum stake.

Lastly - it could be perceived deso keeps tweaking the key "rules" / tokenomics. Further changes ahead as well with proof of stake. Will this reputation ally work in our favour? Or will people see DESO as typical SV break fast mentality?

**Comment by stet** [comment]
this is so funny. but i actually agree.
i also think $deso in reality is only worth less than $10 but this launch sort of bypassed reality thank to the magic of the bonding curve.
but now it is unattractive to all beyond the earliest buyers, which is a known controversy of the project.
